app/assets/javascripts/admin/addon/controllers/admin-dashboard.js

```diff
@@ -49,6 +49,10 @@ export default class AdminDashboardController extends Controller {
   get isReportsTabVisible() {
     return this.visibleTabs.includes("reports");
   }
+  @computed("visibleTabs")
```

Comment:
```suggestion

  @computed("visibleTabs")
```

nit, needs to be a gap between functions